src/SignalR/server/Core/src/HubOptions.cs

```diff
@@ -46,5 +46,10 @@ public class HubOptions
         /// Detailed error messages include details from exceptions thrown on the server.
         /// </summary>
         public bool? EnableDetailedErrors { get; set; } = null;
+
+        /// <summary>
+        /// Gets or sets the max buffer size for client upload streams.
```

Comment:
Should we include the default value in our comments? I see we only did it for `MaximumReceiveMessageSize`, but I think we should be doing it for all of the options.